doc/examples/transform/plot_ransac.py

```diff
@@ -114,3 +117,53 @@
            marker='o', label='Outlier data')
 ax.legend(loc='lower left')
 plt.show()
+
+######################################################################
+# Circle model
+# ============
+from skimage.measure import CircleModel
+
+
+def generate_noisy_circle(size, noise_level, radius=1):
+    angle = 2 * np.pi * np.random.rand(size)
+    rad = radius + noise_level * (2 * np.random.rand(size) - 1)
+    x = rad * np.cos(angle)
+    y = rad * np.sin(angle)
+    return x, y
+
+
+# Generate three types of data points
+x1, y1 = generate_noisy_circle(1000, 0.001)
+x2, y2 = generate_noisy_circle(1000, 0.1)
+half_width = 1.5
+x3, y3 = (2 * half_width * np.random.rand(1000) - half_width,
+		  2 * half_width * np.random.rand(1000) - half_width)
+
+x, y = np.concatenate((x1, x2, x3)), np.concatenate((y1, y2, y3))
+xy = np.stack([x, y], axis=1).astype(float)
+
+# Fit with RANSAC
+ransac_kwarg = {'min_samples': 100,
```

Comment:
I am not familiar with RANSAC, but looking at the output qualitatively I was wondering why only about half of the circle shows up as inliers in the right panel of the plot?

empirically it seems that changing `min_samples` to a value <= 80 helps get many more inliers along the circle boundary.